src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubBuildStatusNotification.java

```diff
@@ -176,18 +178,28 @@ private static GitHubSCMSource getSCMSource(final SCMSourceOwner scmSourceOwner)
      * With this listener one notifies to GitHub when a Job (associated to a PR) has been scheduled.
      * Sends: GHCommitState.PENDING
      */
-    @Extension public static class PRJobScheduledListener extends QueueListener {
+    @Extension
+    public static class PRJobScheduledListener extends QueueListener {
```

Comment:
People complain when I make big sweeping changes with 1000+ line diffs... they can see if they prefer death by 100 x 10 line diffs.

The point of the queue listener is to flag the status as early as possible. When we get the event from GH we do not know if any jobs will be triggered... we need to let the branch-api do its stuff and react to the event... that will queue jobs and then this listener flags the status before the job even starts a checkout